index.html

```diff
@@ -1,8 +1,118 @@
 <!DOCTYPE html>
 <html>
   <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" href="styles/styles.css">
   </head>
-
   <body>
+    <section class="background">
+      <header>
+        <ul>
+          <li>central pennsylvania</li>
+          <li>open source conference</li>
+        </ul>
+      </header>
+        <section class="hero">
+          <h1>Into the Great Wide Open (Source)</h1>
+        </section>
+    </section>
+    <section>
+      <div class="whitesquare">
+        0
+      </div>
+      <div class="whitesquare">
+        0
+      </div>
+      <div class="whitesquare">
+        0
+      </div>
+        <ul>
+          <li>Days until CPOSC 2018</li>
+          <li>November 1, 2018</li>
+        </ul>
+    </section>
+    <section>
+      <p>2017 tickets are now available:</p>
+        <div>
+          buy tickets
+        </div>
+    </section>
+    <section>
+      <h2>The Central PA Open Source Conference (CPOSC) is a one-day technical conference for open source users and developers.</h2>
+      <p>Presentations and sessions cover all aspects of open source, with talks ranging from novice to expert skill levels and featuring case studies, best practices, code-alongs and more. Join us for a day of meeting with and learning from your technology peers.</p>
+    </section>
+    <section>
+      <article>
```

Comment:
Are you creating the background layers for the content? It will make it easier for the both of us if you do have your content layers.